### PR TITLE
created home-page

### PR DESCRIPTION
**Pull request by agrawalshanu708**
Created Ecommerce Home page
Added sticky Navbar
Added a banner
Added shop by categories
added brand sticker
ecom-hp-1
ecom-hp-2

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,41 +1,20 @@
 import "./App.css";
 import logo from "./logo.png";
-
+import { Routes, Route, Link } from "react-router-dom";
+import Mockman from "mockman-js"
+import {Navbar,Footer} from "./components/index"
+import {Home} from "./pages/Home/homepage"
+import {ProductListing} from "./pages/ProductListing/productListing"
```
Comment (harshitpaliwal95) [inline src/App.js:7]: can create same file to import pages

File: src/components/Navbar/Navbar.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
+.navbar-search{
+    padding : 1rem;
+    width: 30rem;
+}
+.navbar>div{
+    /* border: 1px solid black; */
```
Comment (harshitpaliwal95) [inline src/components/Navbar/Navbar.css:6]: remove commented code

File: src/pages/Home/homepage.jsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    <img src= {men3} alt="" />
+    </div>
+    </div>
+
+    const WomenDisplay = () =>   <div className="sliding-cards">
+    <div>
+    <img src={women2} alt="" />
+    </div>
+    <div>
+    <img src= {women1} alt="" />
```
Comment (harshitpaliwal95) [inline src/pages/Home/homepage.jsx:30]: write something in alt tag

**Review by harshitpaliwal95 — COMMENTED** [review]
added some suggestion over all code looks good

File: src/Assets/index.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+
+
+export {
+    banner1,
+    luxury1,luxury2,luxury3,
+    men1,men2,men3,
+    women1,women2,women3,
+    brand1,brand2,brand3,brand4,brand5
+}
```
Comment (Sharmaryan) [inline src/Assets/index.js:25]: it is a good thing that you are using a single file for export

File: src/components/Navbar/Navbar.css (hunk 2 above)
Comment (Sharmaryan) [inline src/components/Navbar/Navbar.css:6]: you can remove it, as it is already commented

File: src/components/Navbar/Navbar.jsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
+import {Link } from "react-router-dom";
+import "./Navbar.css"
+const  Navbar = () => {
+    return (
+        <>
+               <div class="navbar col-12">
+               <div class="nav-logo-box ">
+               <img src = ""/>
```
Comment (Sharmaryan) [inline src/components/Navbar/Navbar.jsx:8]: empty src attribute

File: src/pages/Home/homepage.jsx [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+import {banner1,luxury1,luxury2,luxury3,men1,men2,men3,women1,women2,women3,brand1,brand2,brand3,brand4,brand5} from "../../Assets/index"
+
+
+const Home = () => {
+
+    const[display, setDisplay] = useState("men")
+
+    const MenDisplay = () =>   <div className="sliding-cards">
+    <div>
+    <img src= {men1} alt="" />
```
Comment (Sharmaryan) [inline src/pages/Home/homepage.jsx:13]: add alt text here

File: src/pages/Home/homepage.jsx [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+
+         {/* ---------------------------BRANDS------------------------------ */}
+
+    <h2>Brands</h2>
+     <div className="brand-container">
+         <img src= {brand1} alt="" />
+         <img src={brand2} alt="" />
+         <img src={brand3} alt="" />
+         <img src={brand4} alt="" />
+         <img src={brand5} alt="" />
```
Comment (Sharmaryan) [inline src/pages/Home/homepage.jsx:77]: same here

**Review by Sharmaryan — APPROVED** [review]
suggested minor changes

**Pull request merged**